themes/default/content/registry/packages/azure-native/installation-configuration.md

```diff
@@ -297,10 +297,11 @@ as shown above except use `clientCertPath`/`ARM_CLIENT_CERTIFICATE_PATH` instead
 
 ### Authenticate using Managed Service Identity (MSI)
 
-When MSI is used for authentication, all
-[configuration is done in Azure](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
-and we only need to tell the provider to use MSI. You can set the Pulumi configuration `useMsi` or the environment
-variable `ARM_USE_MSI` to "true".
+MSI is [configured in Azure](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview).
+
+When using a system-assigned identity, or when using a user-assigned identity _and there's only a single one_, you don't need to do anything else. The provider will automatically use the identity.
+
+When one or more resources in your program have multiple user-assigned identities, you need to set the `clientId` config or the `ARM_CLIENT_ID` environment variable to the client ID of the identity you want to use.
```

Comment:
`resources in your program have multiple user-assigned identities` is this something that will be obvious to Azure users? I don't quite follow what it means.

Reply:
I _think_ it should be pretty clear to Azure people. It looks like this in a Pulumi program:
```go
			vmIdentity = &compute.VirtualMachineIdentityArgs{
				Type:                   compute.ResourceIdentityTypeUserAssigned,
				UserAssignedIdentities: pulumi.StringArray{umi.ID(), umi2.ID()},
			}
```

Reply:
Clarification on the notion of "user-assigned" vs "system-assigned" identities here: https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview#managed-identity-types (maybe we should link to this).